source/2019-08-23-the-ember-times-issue-112.md

```diff
@@ -9,18 +9,22 @@ responsive: true
 Gi suilon Emberistas! 🐹
 
 This week:
-watch talks from 1st Chennai EmberJS Meetup 🇮🇳, 
+watch talks from 1st Chennai EmberJS Meetup 🇮🇳,
+announcing the official Ember Octane release plan 🏎🐹,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Octane Is Coming 🏎🐹](https://blog.emberjs.com/2019/08/15/octane-release-plan.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The time has come: [Ember Octane](https://emberjs.com/editions/) is coming to your Ember application very soon!
+As [recently announced in the official Ember Octane release plan](https://blog.emberjs.com/2019/08/15/octane-release-plan.html), the very first edition of Ember is going to be released at the same time as the planned release date of Ember v3.14.
+
+But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane-related features and enable them by default. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation (API Docs and Guides) and migration tools.
```

Comment:
```suggestion
But what does it mean? By the time Ember 3.14 is published, the framework will include all Octane-related features and enable them by default. Additionally, the "Octane way" will be the primary and recommended way to build Ember apps. This will also be reflected in Ember's official documentation ([API Docs](https://api.emberjs.com/ember/release) and [Guides](https://octane-guides-preview.emberjs.com/release/upgrading/editions/)) and migration tools.
```